hackathon_site/registration/jinja2/registration/signup.html

```diff
@@ -0,0 +1,34 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Sign up{% endblock %}
+
+{% block form_img %}<img src="{{ static('event/images/landing.svg') }}"  alt="Signup image" />{% endblock %}
+
+{% set form_action=url("registration:signup") %}
+               
+{% block form_inputs %}
+    {% for field in form %}
+        {{ render_field(field) }}
+    {% endfor %}
+
+    {% if form.non_field_errors %}
+        {% for error in form.non_field_errors() %}
+            {{ error }}
+        {% endfor %}
+    {% endif %}
+{% endblock %}
```

Comment:
Is it possible to add a helper text below a field? I'd like "Must contain at least 8 characters" below the first password input

Reply:
Must contain letters blah blah